tests/system/feature_store/test_feature_store.py

```diff
@@ -2120,11 +2121,18 @@ def test_parquet_target_vector_overwrite(self):
             .equals(dfout2.set_index(keys="name").sort_index())
         )
 
-    def test_overwrite_specified_nosql_path(self):
+    @pytest.mark.parametrize("use_ds_profile", [True, False])
+    def test_overwrite_specified_nosql_path(self, use_ds_profile):
         df1 = pd.DataFrame({"name": ["ABC", "DEF", "GHI"], "value": [1, 2, 3]})
         df2 = pd.DataFrame({"name": ["JKL", "MNO", "PQR"], "value": [4, 5, 6]})
-
-        targets = [NoSqlTarget(path="v3io:///bigdata/overwrite-spec")]
+        if use_ds_profile:
+            profile = DatastoreProfileV3IO(
+                name="v3io_profile", v3io_access_key=os.getenv("V3IO_ACCESS_KEY")
+            )
+            register_temporary_client_datastore_profile(profile)
+            targets = [NoSqlTarget(path="ds://v3io_profile//bigdata/overwrite-spec")]
```

Comment:
```suggestion
            targets = [NoSqlTarget(path="ds://v3io_profile/bigdata/overwrite-spec")]
```

Reply:
On purpose! Its equivalent  of 'v3io:///'. I know it's ugly - any other suggestions are welcomed

Reply:
That's not a reasonable expectation from users. This needs to be moved into the store itself somehow.

Btw, I'm guessing it relates to my comment from before (https://github.com/mlrun/mlrun/pull/5158/files#r1502214646).